models/CoronSegmentator/runtests.sh

```
@@ -0,0 +1,373 @@
+function print_error_msg() {
+    echo "${red}Error: $1.${noColor}"
+    echo ""
+}
+
+function print_style_fail_msg() {
+    echo "${red}Check failed!${noColor}"
+    echo "Please run auto style fixes: ${green}./runtests.sh --autofix${noColor}"
+}
+
+function is_pip_installed() {
+    return $(${PY_EXE} -c "import sys, pkgutil; sys.exit(0 if pkgutil.find_loader(sys.argv[1]) else 1)" $1)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)